src/middlewared/middlewared/plugins/snapshot.py

```diff
@@ -95,11 +95,13 @@ async def do_create(self, data):
         """
         Create a Periodic Snapshot Task
 
-        Create a Periodic Snapshot Task that will take snapshots of specified `dataset` at specified `schedule`.
+        Create a Periodic Snapshot Task that will take snapshots of specified `dataset` or `zvol` at specified `schedule`.
         Recursive snapshots can be created if `recursive` flag is enabled. You can `exclude` specific child datasets
-        from snapshot.
+        or zvols from the snapshot.
         Snapshots will be automatically destroyed after a certain amount of time, specified by
         `lifetime_value` and `lifetime_unit`.
+        If multiple periodic tasks create snapshots at the same time (for example hourly and daily at 00:00) the snapshot
+        will be kept until the last of these tasks reaches its expiry time. 
```

Comment:
There is a whitespace at the end of this line too.

Reply:
Space fixed. As above, can't see but believe its done.